### DISCUSSION TITLE
Plans to target IDF 5.2?

### DISCUSSION THREAD

**Discussion by holdersn** [espressif/arduino-esp32#9420] [Question - Community Help]
With idf-release/v5.1 there is currently an active development branch which targets IDF 5.1

IDF 5.2 has been around for a few weeks. Are there any plans to target IDF 5.2?

If so, …

1. both 5.1 and 5.2, or just 5.2 instead of 5.1?
2. Is there a rough/estimated timeframe when development on a 5.2 branch would start?

**Comment by me-no-dev** [comment]
plan is to move to 5.3 directly. Will start the process after 3.0.0 is released.

**Reply by holdersn** [reply]
ok, thanks!

**Comment by Fyod** [comment]
Is the plan still to Wait for 5.3? 
It would be nice to have multiple TWAI controller support in Arduino now that IDF 5.2 has been stable for quite a while.
Or is there any way to hack multiple TWAI controller support into the current release?

**Reply by me-no-dev** [reply]
We will not wait for 5.3, we will start work on it. 5.2 makes no sense, because P4 is supported in 5.3 and we need to add support for it.

**Comment by Jason2866** [comment]
@Fyod Made a version based and Arduino compiled with IDF 5.2.x. Only useable with Platformio.
You can try with
```
platform = https://github.com/Jason2866/platform-espressif32.git#Arduino/IDF52
```
Zero support, you walk alone when issues arrive ;-)
